Cargo.toml

```diff
@@ -37,19 +38,20 @@ v2_28 = ["v2_26", "webkit2gtk-sys/v2_28"]
 
 [dependencies]
 bitflags = "^1.0"
+once_cell = "1.8"
 libc = "^0.2"
-cairo-rs = "^0.9.0"
-gdk = "^0.13.0"
-gdk-sys = "^0.10.0"
-gio = "^0.9.0"
-gio-sys = "^0.10.0"
-glib = "^0.10.1"
-glib-sys = "^0.10.0"
-gobject-sys = "^0.10.0"
-gtk = "^0.9.0"
-gtk-sys = "^0.10.0"
-javascriptcore-rs = "0.10"
+cairo-rs = "^0.14.0"
+gdk = "^0.14.0"
+gdk-sys = "^0.14.0"
```

Comment:
You don't need the -sys dependencies

Reply:
Not sure what exactly you mean. If I remove all the `-sys` dependencies all that is left is
```
[dependencies]
bitflags = "^1.0"
libc = "0.2"
```

and I'm left with lots of `unresolved import 'gtk_sys'`.